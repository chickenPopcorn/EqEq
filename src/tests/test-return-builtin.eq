SomeContext = {
  a = {  
        cos(42);
      }
  b = {
  		log(42);
  	  }
  c = {
  	    sin(42);
      }
  d = {
  	    tan(42);
      }
  e = {
  	    sqrt(42);
      }
}

SomeContext:find a {
  print("%.2f\n", a);
  print("%.2f\n", b);
  print("%.2f\n", c);
  print("%.2f\n", d);
  print("%.2f\n", e);
}