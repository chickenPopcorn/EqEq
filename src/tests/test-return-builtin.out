-0.40
3.74
-0.92
2.29
6.48
